docs/manage/users-roles/roles/migrate-audit-logging.md

```diff
@@ -0,0 +1,45 @@
+---
+id: migrate-audit-logging
+title: Migrating Audit Logs from Role V1 to V2
```

Comment:
```suggestion
title: Migrating Audit Logs Queries from Role V2 to V2
```
This seems more appropriate

Reply:
@atishya-22, I think you meant to say "...from Role V1 to V2" instead of "...from Role V2 to V2".

Reply:
yes